### PR TITLE
Add docs for Microsoft IIS

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**

### CODE REVIEW

File: docs/xdr/features/collect/integrations/application/microsoft_iis.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+Please read the dedicated documentation for each concentrator:
+
+- [Rsyslog](https://www.notion.so/ingestion_methods/rsyslog/)
+- [Logstash](https://www.notion.so/ingestion_methods/logstash/)
+- [Syslog-ng](https://www.notion.so/ingestion_methods/syslog-ng/)
+- [Graylog](https://www.notion.so/ingestion_methods/graylog/)
+- [Sekoia.io docker concentrator](https://www.notion.so/ingestion_methods/sekoiaio_forwarder/)
+
+!!! Note
+While [Sekoia.io](http://sekoia.io/) docker concentrator is highly recommended, you are free to use the one that you are most comfortable with.
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/application/microsoft_iis.md:201]:
```suggestion
    While [Sekoia.io](http://sekoia.io/) docker concentrator is highly recommended, you are free to use the one that you are most comfortable with.
```

File: docs/xdr/features/collect/integrations/application/microsoft_iis.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+To ensure proper configuration, it is important to replace the `RSYSLOG_HOST` variable with the IP address of your concentrator.
+
+!!! Warning
+`OutputType Syslog_TLS` is needed for `TCP` transport even if you do not encrypt data. It does not depend on SSL transport at all.
+**Remove it ONLY** if you use `UDP` - `om_udp`.
+For more information, consult [NXLog documentation.](https://docs.nxlog.co/refman/current/xm/syslog.html)
+
+!!! Note
+The iso8859-1 character encoding is limited to 256 characters, which is not enough to represent all French characters. This means that some French characters might not be correctly interpreted or displayed when using iso8859-1 encoding. For example, iso8859-1 does not include characters such as é, è, ê, and ë.
+In order to correctly represent these characters, it is recommended to install the [Sekoia.io agent](https://docs.sekoia.io/xdr/features/collect/integrations/endpoint/sekoiaio/). This endpoint agent is specifically designed to handle such issues, ensuring the accurate and secure transmission of data.
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/application/microsoft_iis.md:181]:
```suggestion
    The iso8859-1 character encoding is limited to 256 characters, which is not enough to represent all French characters. This means that some French characters might not be correctly interpreted or displayed when using iso8859-1 encoding. For example, iso8859-1 does not include characters such as é, è, ê, and ë.
    In order to correctly represent these characters, it is recommended to install the [Sekoia.io agent](https://docs.sekoia.io/xdr/features/collect/integrations/endpoint/sekoiaio/). This endpoint agent is specifically designed to handle such issues, ensuring the accurate and secure transmission of data.
```

File: docs/xdr/features/collect/integrations/application/microsoft_iis.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+```
+
+!!! Info
+To ensure proper configuration, it is important to replace the `RSYSLOG_HOST` variable with the IP address of your concentrator.
+
+!!! Warning
+`OutputType Syslog_TLS` is needed for `TCP` transport even if you do not encrypt data. It does not depend on SSL transport at all.
+**Remove it ONLY** if you use `UDP` - `om_udp`.
+For more information, consult [NXLog documentation.](https://docs.nxlog.co/refman/current/xm/syslog.html)
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/application/microsoft_iis.md:177]:
```suggestion
    `OutputType Syslog_TLS` is needed for `TCP` transport even if you do not encrypt data. It does not depend on SSL transport at all.
    **Remove it ONLY** if you use `UDP` - `om_udp`.
    For more information, consult [NXLog documentation.](https://docs.nxlog.co/refman/current/xm/syslog.html)
```

File: docs/xdr/features/collect/integrations/application/microsoft_iis.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+  Exec to_syslog_ietf();
+</Output>
+<Route iis_to_rsyslog>
+  Path iis => rsyslog
+</Route>
+
+```
+
+!!! Info
+To ensure proper configuration, it is important to replace the `RSYSLOG_HOST` variable with the IP address of your concentrator.
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/application/microsoft_iis.md:172]:
```suggestion
    To ensure proper configuration, it is important to replace the `RSYSLOG_HOST` variable with the IP address of your concentrator.
```

**Review by squioc — CHANGES_REQUESTED** [review]
Thanks for the work.
Tabs are missing for notes and warnings that break the layout of the page. Can you fix it?

**Review by squioc — CHANGES_REQUESTED** [review]
Two little changes to apply.

File: docs/xdr/features/collect/integrations/application/microsoft_iis.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+</Route>
+
+```
+
+!!! Info
+    To ensure proper configuration, it is important to replace the `RSYSLOG_HOST` variable with the IP address of your concentrator.
+
+!!! Warning
+    `OutputType Syslog_TLS` is needed for `TCP` transport even if you do not encrypt data. It does not depend on SSL transport at all.
+    **Remove it ONLY** if you use `UDP` - `om_udp`.
```
[the remark this answers is not in the document]
Comment (lvoloshyn-sekoia) [reply docs/xdr/features/collect/integrations/application/microsoft_iis.md:176]: It seems to be connected, though: `Module om_udp` vs `Module om_tcp`. Perhaps, a mistake in the original doc?

**Review by squioc — APPROVED** [review]

**Pull request merged**